language/English/strings.po

```diff
@@ -8342,7 +8346,11 @@ msgctxt "#19178"
 msgid "Show channel information when switching channels"
 msgstr ""
 
-#empty string with id 19179
+#. Used as extension header on recordings window
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19179"
+msgid "Deleted recordings trash"
```

Comment:
Thanks, in that case "Deleted" as the window name I think would be enough as you have the context of "TV - Recordings - Deleted" displayed.

@da-anda for a 2nd opinion